### ISSUE TITLE
`bundle install --standalone` produces a `setup.rb` file that does not setup load paths on 1.8.7 using bundler 1.6.2

### ISSUE THREAD

**Issue by myronmarston** [bundler/bundler#3004]
I've put together a gist with a repro script:

https://gist.github.com/myronmarston/5a93e0ed953ba0bb9226

The output I'm getting is shown there.

Here's the `bundle/bundler/setup.rb` file produced:

``` ruby
require 'rbconfig'
# ruby 1.8.7 doesn't define RUBY_ENGINE
ruby_engine = defined?(RUBY_ENGINE) ? RUBY_ENGINE : 'ruby'
ruby_version = RbConfig::CONFIG["ruby_version"]
path = File.expand_path('..', __FILE__)
```

Notice that it does add anything to `$LOAD_PATH` like it's supposed to.  That causes `require 'diff/lcs'` to fail.  I've noticed that if I rerun `bundle install --standalone`, it seems to work...so there's something different about when it's run the 1st time vs the Nth time.

This may be related to #2851, but I'm not sure about that yet.

**Comment by toppy42** [comment]
@myronmarston Just banged my head against this for an hour.  Thanks for the explanation.

**Comment by segiddins** [comment]
This appears to be fixed as of bundler 1.9.9, as the repro script provided now passes.
